corehq/apps/app_manager/views/multimedia.py

```diff
@@ -37,7 +39,12 @@ def multimedia_list_download(request, domain, app_id):
 def multimedia_ajax(request, domain, app_id):
     app = get_app(domain, app_id)
     if app.get_doc_type() == 'Application':
-        multimedia_state = app.check_media_state()
+        try:
+            multimedia_state = app.check_media_state()
+        except ReportConfigurationNotFoundError:
+            return JsonResponse(
+                {"message": _("One of the Report menus is misconfigured, please try again after they are fixed")},
```

Comment:
Is this a server error or a user error? It doesn't necessarily read as an error that the user can do anything about.

Also seems a "NotFoundError" should be a 404 or something

Reply:
There is more context in the ticket, but it's a configuration issue caused partly by the user and partly by the way the feature is setup. If the user navigates to one of the reporting menu (hopefully the message here conveys that), the user would see an instruction of what exactly is the misconfiguration and a step to fix it. It's difficult to capture the exact issue and display it here. Plus the JS/UI side is setup to easily handle this as an error.